src/common/Constants.cpp

```diff
@@ -78,6 +78,11 @@ unsigned int ReadGpuConstants(std::string propertyName) {
   return pt.get<unsigned int>("node.gpu." + propertyName);
 }
 
+std::string ReadGpuVariable(std::string propertyName) {
```

Comment:
Can you change the parameter to `const std::string&` and the function name to `ReadGPUVariableFromConstantsFile`?